docs/static/security/users.asciidoc

```diff
@@ -1,47 +1,76 @@
 [role="xpack"]
-[[ls-security]]
-=== Configuring Security in Logstash
+[[ls-grant-access]]
+=== Grant users access to secured resources
 [subs="attributes"]
-++++
-<titleabbrev>Configuring Security</titleabbrev>
-++++
 
-The Logstash {es} plugins ({logstash-ref}/plugins-outputs-elasticsearch.html[output],
-{logstash-ref}/plugins-inputs-elasticsearch.html[input],
-{logstash-ref}/plugins-filters-elasticsearch.html[filter]
-and {logstash-ref}/monitoring-logstash.html[monitoring])
-support authentication and encryption over HTTP.
+You can use role-based access control to grant users access to secured
+resources. The roles that you set up depend on your organization's security
+requirements and the minimum privileges required to use specific features.
 
-To use Logstash with a secured cluster, you need to configure authentication
-credentials for Logstash. Logstash throws an exception and the processing
-pipeline is halted if authentication fails.
+Typically you need the create the following roles:
 
-If encryption is enabled on the cluster, you also need to enable TLS/SSL in the
-Logstash configuration.
+* <<create-writer-role,writer role>> for publishing events
+* <<create-reader-role,reader role>> so that users can access the indices {ls} creates
 
-If you want to monitor your Logstash instance with {monitoring}, and store the
-monitoring data in a secured {es} cluster, you must configure Logstash
-with a username and password for a user with the appropriate permissions.
+{es-security-features} provide {ref}/built-in-roles.html[built-in roles] that
+grant a subset of the privileges needed by {ls} users. 
+For example, `logstash_admin` is a built-in role that provides access to `.logstash-*`
+indices for managing configurations.
+
+When possible, use the built-in roles to minimize the affect of future changes
+on your security strategy.
+
+NOTE: You can assign roles and privileges to **API keys** as an alternative to
+using usernames and passwords. See <<ls-api-keys>> for more information on that
+alternate approach to security.
```

Comment:
Which is the preferred method, and which is simplest? I think that part of the confusion with configuring security is providing users with every option. Instead, can we present either RBAC or API keys as the preferred method and then link off to the other method? I think that a defined workflow would help to orient users.